interface/super/edit_list.php

```diff
@@ -1036,7 +1036,7 @@ function mysubmit() {
             <div class="collapse navbar-collapse" id="navbar-list">
                 <ul class="nav navbar-nav">
                     <li class="nav-item"><a class="nav-link" href="#" data-toggle="modal" data-target="#modal-new-list"><i class="fa fa-plus"></i>&nbsp;<?php echo xlt('New List'); ?></a></li>
-                    <li class="nav-item"><a class="nav-link" href="#" class="deletelist" id="<?php echo attr($list_id); ?>"><i class="fa fa-trash"></i>&nbsp;<?php echo xlt('Delete List'); ?></a></li>
+                    <li class="nav-item"><a class="nav-link deletelist" href="#" id="<?php echo attr($list_id); ?>"><i class="fa fa-trash"></i>&nbsp;<?php echo xlt('Delete List'); ?></a></li>
```

Comment:
@tywrenn Any way to add a check for this error case? I've fixed several of these and they tend to be in our li updates.

Reply:
Haven't found a linter yet that finds this -- this is a very easy mistake to have.

Reply:
Haven't found a linter yet that finds this -- this is a very easy mistake to have.